packages/database/src/entities/ExpoJudgingSessionContext.ts

```diff
@@ -10,7 +10,7 @@ type ConstructorArgs = Pick<ExpoJudgingSessionContext, 'judge' | 'expoJudgingSes
 @Entity()
 @Unique({ properties: ['judge', 'expoJudgingSession'] })
 export class ExpoJudgingSessionContext extends Node<ExpoJudgingSessionContext> {
-  @OneToOne({ entity: () => Judge, ref: true, unique: false })
+  @ManyToOne({ entity: () => Judge, ref: true, unique: false })
```

Comment:
shouldn't it be many to one since one judge can have multiple expo judging session?

Reply:
Yep, that's why it's being changed to `ManyToOne`; `OneToOne` was wrong

Reply:
my bad i read that as Many to Many idk why :/